jobs/build.Jenkinsfile

```diff
@@ -552,3 +557,22 @@ def run_release_job(buildID) {
         ]
     }
 }
+
+// XXX: generalize and put in coreos-ci-lib eventually
+def wait_until_locked_or_continue(resource, timeout_mins) {
```

Comment:
maybe `wait_until_locked_or_timeout`?

Reply:
or `wait_until_locked_with_timeout`.. I'm not crazy about any of these. They're just suggestions.

Reply:
Though `timeout` makes it sound like it will error out. I like that `continue` communicates that it's non-fatal.